great_expectations/checkpoint/actions.py

```diff
@@ -50,6 +51,10 @@
 from great_expectations.data_context.util import instantiate_class_from_config
 from great_expectations.datasource.fluent.config_str import ConfigStr
 from great_expectations.exceptions import ClassInstantiationError
+from great_expectations.exceptions.exceptions import (
+    ValidationActionAlreadyRegisteredError,
+    ValidationActionRegistryRetrievalError,
```

Comment:
Do we need custom error types? Ruff says so with our rule we've opted in and it's a low lift.